.github/workflows/ci.yml

```diff
@@ -4,31 +4,41 @@ on:
   pull_request:
   push:
     branches:
-    - staging
-    - trying
+    - rsk_merge_master_Dec2021
 
 jobs:
   lint:
-    runs-on: [self-hosted, CI-worker]
+    runs-on:  ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: setup-env
+      - name: modify rsk node conf and install mkcert
         run: |
-          echo ZKSYNC_HOME=$(pwd) >> $GITHUB_ENV
-          echo $(pwd)/bin >> $GITHUB_PATH
-
+          sed -i '/autoMine/ s/true/false/' docker/rskj/node.conf
```

Comment:
why do we need to set `automine` false  to run the lint job?

Reply:
I think we don't need it, I just copy the configuration from ci-iov-integration.yml to make it run